src/main/java/com/gregtechceu/gtceu/common/data/GTRecipes.java

```diff
@@ -13,9 +13,12 @@
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.world.level.block.ComposterBlock;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.function.Consumer;
 
 public class GTRecipes {
+    private static final List<ResourceLocation> RECIPE_FILTERS = new ArrayList<>();
```

Comment:
Use a set.

Reply:
Migrated to ObjectOpenHashSet, since it has a smaller memory footprint than the default HashSet.